Oceanbase integration causes issues with other databases

When using Flyway with the flyway-database-oceanbase jar available in the Command Line and using a database like PostgreSQL, the query to check if the DatabaseType is correct is throwing the following error:

```SQL
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at or near "LIKE"
  Position: 16
```

This is caused by the query in [handlesDatabaseProductNameAndVersion](https://github.com/flyway/flyway-community-db-support/blob/main/flyway-database-oceanbase/src/main/java/org/flywaydb/community/database/oceanbase/OceanBaseDatabaseType.java#L81) which fails if the Database does not support that syntax. Unfortunately, for some databases, such as PostgreSQL, if a statment fails within a transaction then the whole transaction is marked as failed and errors any following statements.

--- a/flyway-database-oceanbase/src/main/java/org/flywaydb/community/database/oceanbase/OceanBaseDatabaseType.java
+++ b/flyway-database-oceanbase/src/main/java/org/flywaydb/community/database/oceanbase/OceanBaseDatabaseType.java
@@ -79,6 +79,10 @@ public class OceanBaseDatabaseType extends BaseDatabaseType {
 
     @Override
     public boolean handlesDatabaseProductNameAndVersion(String databaseProductName, String databaseProductVersion, Connection connection) {
+        if (!databaseProductName.contains("MySQL")) {
+            return false;
+        }
+
         String versionComment;
         try {
             versionComment = OceanBaseJdbcUtils.getVersionComment(connection);

